Gruntfile.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function(grunt) {
+   grunt.initConfig({
+       jshint: {
+          files: ['Gruntfile.js', 'test/**/*.js', 'lib/*.js', '*.js']
+        },
+       mochaTest: {
+         test: {
+           options: {
+             mocha: require('mocha'),
+             reporter: 'spec',
+             timeout: 2000
+           },
+           src: ['tests/*.js']
+         }
+       }});
+    grunt.loadNpmTasks('grunt-contrib-jshint');
+    grunt.loadNpmTasks('grunt-mocha-test');
+
+    grunt.registerTask('default', ['jshint', 'mochaTest']);
+    };
```

Comment:
Several lines here are not aligned correctly and makes the file hard to read. This close }; should match the indentation of the module.exports line. The grunt.loadNpmTasks lines should match the indentation of the grunt.initConfig line. The closing }}); before the first grunt.loadNpmTests line should match the indentation of the grunt.initConfig line as well.